browser/net/brave_network_delegate_base.cc

```diff
@@ -27,13 +28,13 @@ using net::URLRequest;
 namespace {
 
   content::WebContents* GetWebContentsFromProcessAndFrameId(
-      int render_process_id, int render_frame_id) {
+      int render_process_id, int render_frame_id, int frame_tree_node_id) {
   if (render_process_id) {
     content::RenderFrameHost* rfh =
         content::RenderFrameHost::FromID(render_process_id, render_frame_id);
     return content::WebContents::FromRenderFrameHost(rfh);
   }
-  return content::WebContents::FromFrameTreeNodeId(render_frame_id);
+  return content::WebContents::FromFrameTreeNodeId(frame_tree_node_id);
```

Comment:
typo

Reply:
This is causing a couple dozen browser_tests to fail:
`npm run test -- brave_browser_tests`

I noticed chrome/browser/chrome_content_browser_client.cc uses the render_frame_id directly and not the frame_tree_node_id too.

Reply:
reverted and filed https://github.com/brave/brave-browser/issues/2263